data/src/main/java/io/gnosis/data/repositories/EnsRepository.kt

```diff
@@ -50,7 +50,8 @@ class EnsRepository(
 
 
     companion object {
-        private val ENS_ADDRESS = BuildConfig.ENS_REGISTRY.asEthereumAddress()!!
+
+        val ENS_ADDRESS = BuildConfig.ENS_REGISTRY.asEthereumAddress()!!
```

Comment:
You still seem to have a this constant defined in the SettingsFragment or somewhere in the UI level if I remember correctly. Is that really necessary?

Reply:
Just a reference constant AdvancedAppSettingFragment for readability sake:
 val ENS_REGISTRY = EnsRepository.ENS_ADDRESS